eduNEXT/xblock-mindmap
# fix: JavaScript translations

### Description
This PR adds all configurations to use JavaScript translations.

### How To Test
1. Install this branch in your environment.
2. Change the platform language in `(Account → Site Preferences → Site Language)`. Set it to "Spanish (Latin America)"
3. Add a block in a unit from Studio.
4. You should see all the text of the block in the selected language.

### Using translation commands
1. Run `make extract_translations`
2. Add the translations in each `text.po` file
3. Run `make compile_translations`

### Screenshots
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/294309fc-3cf9-4e8f-a9da-2a7db009e024)
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/c4dc5b32-cc0b-452d-8b4e-c2915a472d4f)
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/ca7f3572-d819-4b2d-bd17-4bf053b1b3e6)
